contrib/test/integration/e2e.yml

```diff
@@ -1,5 +1,8 @@
 ---
 
+- name: Set SELinux label for cri-o binary
+  command: chcon system_u:object_r:container_runtime_exec_t:s0 /usr/local/bin/crio
```

Comment:
``when: ansible_distribution == 'RedHat'``

maybe?

Reply:
This label should work on both platforms.  Are we getting an error when setting it?

Reply:
It looks like the chcon did not fail?